lib/server.js

```diff
@@ -889,73 +885,18 @@ Server.prototype._run = function _run(req, res, route, chain, cb) {
 
         if (arg) {
             if (arg instanceof Error) {
```

Comment:
Not part of this PR, but doesnt `arg instanceof Error === true` imply `arg`? This second check seems redundant and adds a level of indentation. Same with `typeof`